ssl-impl/src/test/java/io/crate/protocols/postgres/ssl/SslConfigurationTest.java

```diff
@@ -87,7 +87,6 @@ public void testTrustStoreLoadingFail() throws Exception {
         settingsBuilder.put(SslConfigSettings.SSL_TRUSTSTORE_PASSWORD_SETTING_NAME, "wrongpassword");
 
         TrustStoreSettings.tryLoad(settingsBuilder.build());
-        fail();
```

Comment:
fail() is not needed since the expectedException will fail the test anyway if no exception is thrown.